docs/examples.rst

```diff
@@ -471,3 +471,38 @@ like so:
 .. include::  ../tests/core/contracts/test_contract_example.py
     :code: python
     :start-line: 1
+    
+Using Infura Rinkeby Node
+=========================
+Import your required libraries
+
+.. code-block:: python
+
+    from web3 import Web3, HTTPProvider
+    
+Intialize your infura node in a variable
+
+.. code-block:: python
+    
+    w3 = Web3(Web3.HTTPProvider("https://rinkeby.infura.io/v3/YOUR_INFURA_KEY"))
+
+ 
+Inject the middle-ware in the variable holding your infura node;
+
+.. code-block:: python
+    
+    w3.middleware_stack.inject(geth_poa_middleware, layer=0)
+    
+Just remember that you have to sign all transactions locally, as infura does not handle any keys from your wallet
+
+.. code-block:: python
+    
+    transaction = contract.functions.function_Name(params)
```

Comment:
I think you're missing a `.transact()` on the end of this line

Reply:
ah ... actually, i'm missing a `.buildTransaction()` at the end

`.transact()` threw an error, i don't remember what but i can update you in around 12 hours from now ( it'll be morning here, i'll be at work where i can review and reproduce it )

that error is one of the reasons i decided to go ahead with posting the whole thing :)